src/pactflow/client/prompt-utils.ts

```diff
@@ -36,7 +36,13 @@ export async function getOADMatcherRecommendations(
   });
 
   const regex = /```json[c5]?([\s\S]*?)```/i;
-  const match = regex.exec(matcherResponse.content.text as string);
+  const content = matcherResponse.content;
```

Comment:
@JP-Ellis – I had to make some changes here to get it to compile. Could you take a look at this?

Reply:
i have verified the changes, as these were added by me for oad recommendations feature